server.js

```diff
@@ -23,8 +15,59 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('', (req, res) => {
+  res.end('Learn about avocado sales. Jeeiij')
+})
+
+// /sales - all sales everywhere
+// /sales?totalBagsSoldMoreThan=1234 - all sales with more than 1234 sold bags
+app.get('/sales', (req, res) => {
+  const totalBagsSoldMoreThan = req.query.totalBagsSoldMoreThan
+  if (totalBagsSoldMoreThan) {
+    const matchingSales = sales.filter(
+      (sale) => sale.totalBagsSold >= totalBagsSoldMoreThan
+    )
+    res.json(matchingSales)
+  } else {
+    res.json(sales)
+  }
+})
+
+// average price, sorted from low to high
+app.get('/salesByAveragePrice', (req, res) => {
```

Comment:
It would be nice if this was part of the `/sales` endpoint (and it'd be more RESTful) - then make use of query params to filter on price. So you could have `/sales` to return all sales and then `/sales?averagePrice=12` for filtering on price